pkg/virt-handler/cmd-client/client.go

```diff
@@ -559,7 +560,31 @@ func (c *VirtLauncherClient) GetDomain() (*api.Domain, bool, error) {
 	}
 	return domain, exists, nil
 }
+func (c *VirtLauncherClient) GetQemuVersion() (string, error) {
+	var qemuVersion string
 
+	request := &cmdv1.EmptyRequest{}
+	ctx, cancel := context.WithTimeout(context.Background(), shortTimeout)
+	defer cancel()
+
+	versionResponse, err := c.v1client.GetQemuVersion(ctx, request)
+	var response *cmdv1.Response
+	if versionResponse != nil {
+		response = versionResponse.Response
+	}
+
+	if err = handleError(err, "GetQemuVersion", response); err != nil {
+		return qemuVersion, err
+	}
+	if versionResponse.Version != "" {
+		if err := json.Unmarshal([]byte(versionResponse.Version), &qemuVersion); err != nil {
```

Comment:
Isn't this just a string? Why does it need to by unmarshalled by json?

Reply:
Same as above.